Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -164,12 +167,15 @@ namespace PhysX
         const auto endPosition = basis.m_controller->GetPosition();
         const auto gravity = basis.m_testScene->GetGravity();
         // The actual distance fallen is quadratic and the relative error is about equal to the timestep divided by the total time.
-        const auto relativeError = (1.0f / 60.0f) / (float(timeStepCount) / 60.0f);
+        const auto relativeError = timeStep / totalTime;
         // calculate distance fallen (d = 0.5 * g * t^2)
         const float relativeDistanceFell = 0.5f * gravity.GetZ() * (totalTime * totalTime);
         const float calculatedDistanceFell = relativeDistanceFell - (relativeDistanceFell * relativeError);
 
-        EXPECT_THAT(endPosition.GetZ(), testing::FloatNear((startPosition.GetZ() + calculatedDistanceFell), 0.0001f));
+        EXPECT_THAT(endPosition.GetZ(), testing::FloatNear((startPosition.GetZ() + calculatedDistanceFell), 0.001f));
     }
 
+    // Indicate the values to call all the tests that use PhysXDefaultWorldTestWithParamFixture.
```

Comment:
nit: Please remove the comment, I put them in the feedback to remark the changes for you :)

Reply:
Removed